README.md

```diff
@@ -200,6 +226,34 @@ docker run -e AWS_ACCESS_KEY_ID='<aws-access-key>' \
 
 Finally, trigger a workflow. You can do this from the Cumulus dashboard, the Cumulus API, or with the AWS Console by supplying a 
 
+## Debugging:
+
+SSH into the ECS container instance.
+
+Make sure the EC2 instance has internet access and is able to pull the image from docker hub by doing:
+
+```
+docker pull cumuluss/cumulus-ecs-task:1.1.1
+```
+
+`cat` the ecs config file to make sure credentials are correct:
+
+```
+cat /etc/ecs/ecs.config
+```
+
+Check if there's multiple entries of the config.
+
+If there is, there are two things to try:
+
+- delete the ec2 instance and redeploy
+- delete the incorrect config and restart the ecs agent (i haven't tested this much but i expect it to work. you'll still want to update the docker credentials in the deployment's app directory). Restart the agent by doing:
+  ```sh
+  sudo stop ecs
+  source /etc/ecs/ecs.config
+  sudo start ecs
+  ```
+
 ## Create a release
 
 Bump the version
```

Comment:
Sorry I know you didn't update this part, but can you make this something like 'Bump the version number and the changes will automatically be released upon merge to master' (if that's even true, haha, idk). Also write something about creating a changelog entry for the release.